canopy_dispatch.ml

```diff
@@ -66,67 +66,72 @@ module Make (S: Cohttp_lwt.Server) (C: V1_LWT.CONSOLE) (Disk: V1_LWT.KV_RO)
     | uri::[] when uri = config.Canopy_config.push_hook_path ->
       store.update () >>= fun l ->
       respond_update l
+    | "tags"::[] -> (
+      let tags = Canopy_content.tags !cache in
+      let content = Canopy_article.to_tyxml_tags tags in
+      respond_html ~headers ~title:config.Canopy_config.blog_name ~content ~updated
```

Comment:
yep, that's an conservative approximation and will result in more data being transferred in the worst case than needed... but better than an underapproximation which leads to outdated data on the clients.. ;)  a new article might as well introduce new tags, as might a change to an existing article.. the logic how to get the last update to any tag (add/remove) is non-trivial